examples/binary/storage/StorageObjectData-complex.json

```diff
@@ -0,0 +1,31 @@
+{
+  "bucket": "sample-bucket",
+  "cacheControl": "in nostrud dolore enim adipisicing",
```

Comment:
Again, this isn't nearly as helpful as seeing a genuine value - likewise for content disposition, customer encryption etc.